generators/app/index.js

```diff
@@ -193,7 +193,17 @@ module.exports = generators.Base.extend({
   writing: function () {
     // Re-read the content at this point because a composed generator might modify it.
     var currentPkg = this.fs.readJSON(this.destinationPath('package.json'), {});
-    var pkg = {
+
+    // Combine the keywords
+    currentPkg.keywords = currentPkg.keywords || [];
+    this.props.keywords = this.props.keywords || [];
+    this.props.keywords.forEach(function (keyword) {
```

Comment:
`this.props.keywords` needs to be initialized first or else it complains about the method not be available.